File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Features/WindowTypeForm.cs
#region Using

using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;

using Gizmox.WebGUI.Common;
using Gizmox.WebGUI.Forms;

#endregion

namespace CompanionKit.Controls.Form.Features
{
    public partial class WindowTypeForm : Gizmox.WebGUI.Forms.Form
    {
        /// <summary>
        /// Creates the From type.
        /// </summary>
        /// <param name="_windowType">Name of the Form type</param>
        public WindowTypeForm(string _windowType)
        {
            InitializeComponent();
            // Set name of the form and text for form label.
            this.Text = string.Format("This is {0} Form.", _windowType);
            this.mobjLabel.Text = string.Format("This is {0} Form. Click on button for closing it.", _windowType);
        }

        /// <summary>
        /// Handles Click event of the 'Close' button.
        /// Closes the demonstrated form.
        /// </summary>
        private void mobjCloseButton_Click(object sender, EventArgs e)
        {
            this.Close();
        }
    }
}

File: Sources 4.5.2/Gizmox.WebGUI.Forms.CompanionKit/Controls/Form/Features/WindowTypeForm.vb
Namespace CompanionKit.Controls.Form.Features

	Public Class WindowTypeForm

        ''' <summary>
        ''' Creates the From type.
        ''' </summary>
        ''' <param name="windowType">Name of the Form type</param>
        Public Sub New(ByVal _windowType As String)

            ' This call is required by the Visual WebGui UserControl Designer.
            InitializeComponent()

            ' Set name of the form and text for form label.
            Me.Text = String.Format("This is {0} Form.", _windowType)
            Me.mobjLabel.Text = String.Format("This is {0} Form. Click on button for closing it.", _windowType)
        End Sub

        ''' <summary>
        ''' Handles Click event of the 'Close' button.
        ''' Closes the demonstrated form.
        ''' </summary>
        Private Sub mobjCloseButton_Click(sender As Object, e As EventArgs) Handles mobjCloseButton.Click
            Me.Close()
        End Sub
	End Class

End Namespace
